dotcom-rendering/src/lib/getAbUrlHash.ts

```diff
@@ -21,3 +21,25 @@ export const getForcedParticipationsFromUrl = (
 
 	return {};
 };
+
+export const getForcedParticipationsFromUrlFlat = (
+	windowHash: string,
+): Record<string, string> => {
+	if (windowHash.startsWith('#ab')) {
+		const tokens = windowHash.replace('#ab-', '').split(',');
+		return tokens.reduce<Record<string, string>>((obj, token) => {
+			const [testId, variantId] = token.split('=');
+
+			if (testId && variantId) {
+				return {
```

Comment:
OK I see the difference between this function `getForcedParticipationsFromUrlFlat` and `getForcedParticipationsFromUrl` above is the shape of the return type, other than that the logic all looks the same, so you could abstract that common code into a single function to be used by these 2 functions.